src/styles/_footer.scss

```diff
@@ -83,7 +75,18 @@
 /* || DESKTOP */
 
 @media (min-width: units('desktop')) {
-  .footer {
+  .footer__return-to-top-feedback {
+    display: flex;
+    justify-content: center;
+    .usa-footer__return-to-top {
+      align-self: flex-end;
+    }
+    .footer__feedback {
+      padding: 0 300px 0 158px;
```

Comment:
Work in progress - need to find better way to make sure feedback module is justified center, while return to top is aligned bottom and to the left

Reply:
Is this still a work in progress?

Reply:
Yeah - I added it to the new task for this week's sprint